### DISCUSSION TITLE
Optional to present collection as a singleton if a user only has access to one item

### DISCUSSION THREAD

**Discussion by josephgeis** [directus/directus#4209] [Feature Requests]
I would like to assign certain users each their own item (e.g. a profile page) that they can edit, but would like it if the admin bypassed the page listing out multiple items if the user was only assigned one item in that collection. I think it would be better if it was part of the role permissions so that some users who are whole-site admins can still manage all the pages (i.e. still have the collection table view), even if there is just one item in the collection.

**Comment by Dupflo** [comment]
Looking for the exact same behavior

**Comment by joselcvarela** [comment]
I believe this could be part of Presets, instead of having custom logic to handle a single item.

**Comment by mmmaxxxe** [comment]
Looking for a very same feature in related tables. Not only based on user but on a specific related ID.
For example "file" that can have a singleton for a related table A, or B or C...etc... these table can be linked to files collection via the file ID

**Reply by anlexN** [reply]
what is `singleton` meaning? official docs have no explanation.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
